packages/tasit-identity-contract/src/GnosisSafe.test.js

```diff
@@ -5,6 +5,8 @@ const { NFT } = ERC721;
 import GnosisSafe from "./GnosisSafe";
 import { local as localAddresses } from "../../tasit-contracts/3rd-parties/gnosis/addresses";
 
+import Account from "../../tasit-account/dist/index";
```

Comment:
`index` should be unnecessary, right?

Reply:
Yes, it is. I've changed that on one of the latest PRs.

Reply:
Perfect, thanks. Resolving